plotly/tests/test_core/test_plotly/test_plot.py

```diff
@@ -9,70 +9,63 @@
 
 import requests
 import six
+import json
```

Comment:
:cow2: we should order the built-in `json` import above the other imports:

https://github.com/plotly/dev-docs/blob/master/reference/styleguides/python.md#imports